### PR TITLE
[FLOC-1369] Make build failures on master noisy.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1369

### CODE REVIEW

File: flocker_bb/zulip_status.py [hunk 1]
```diff
@@ -109,23 +121,61 @@ def _sendMessage(self, (subjects, message)):
...
+
+        if not subjects:
+            # Not on master
+            return
+
+        buildURL = self.parent.getURLForThing(build)
+
+        message = dedent(
+            """
+            :fire: :fire: :fire:
```
Comment (itamarst) [inline flocker_bb/zulip_status.py]: Resisting the urge to bikeshed emoji...

Comment (itamarst) [inline flocker_bb/zulip_status.py]: Maybe add a `@engineering`?

**Comment by itamarst** [comment]
So this looks OK... but I'm wondering about testing it. If you haven't done so yet, maybe run a variant on `staging` with branch other than `master` temporarily edited in?

**Comment by itamarst** [comment]
Ah, you had a comment in JIRA doing exactly that. Excellent.

**Comment by itamarst** [comment]
OK, one comment, then merge.

File: flocker_bb/zulip_status.py (hunk 1 above)
Comment (tomprince) [inline flocker_bb/zulip_status.py]: Feel free to bikeshed.

**Pull request merged**